crates/bevy_window/src/window.rs

```diff
@@ -370,6 +371,16 @@ impl Window {
             .push(WindowCommand::SetPosition { position });
     }
 
+    /// Modifies the position of the window to be in the center of the current monitor
+    ///
+    /// # Platform-specific
+    /// - iOS: Can only be called on the main thread.
+    /// - Web / Android / Wayland: Unsupported.
+    #[inline]
+    pub fn center_window(&mut self) {
+        self.command_queue.push(WindowCommand::Center);
+    }
+
```

Comment:
I don't think this should be a new method, you could change the existing `set_position` to take a `WindowPosition`. It would support all possible extensions like if we add `WindowPosition::BottomRight`... but then you could call `set_position(WindowPosition::Automatic)` which doesn't make sense...

What do you think?

Reply:
@mockersf Interesting idea suggesting other places to put the window except in the center.

Reply:
I considered the fact that we might want to add more positions, such as bottom-right etc., but I wasn't sure if there would be a use case for that and where do we draw the line of being a window manager.  
But if you expect that there would be a use case for more values, then changing `set_position` signature makes sense.
To avoid code duplication (between `set_position` and `create_window`) we would need to abstract out `WindowPosition -> actual position` code, right? So perhaps a function that takes `WindowPosition`, window size and screen size and returns screen space position?